pkg/route/routes_test.go

```diff
@@ -736,16 +736,16 @@ func checkUnusedParamValues(t *testing.T, ctx *app.RequestContext, expectParam m
 }
 
 var handlerFunc = func(route string) app.HandlersChain {
-	return app.HandlersChain{app.HandlerFunc(func(c context.Context, ctx *app.RequestContext) {
```

Comment:
@safeoy   Inaccurate title
```
build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
ci: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
docs: Documentation only changes
feat: A new feature
optimize: A new optimization
fix: A bug fix
perf: A code change that improves performance
refactor: A code change that neither fixes a bug nor adds a feature
style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
test: Adding missing tests or correcting existing tests
chore: Changes to the build process or auxiliary tools and libraries such as documentation generation
```

content refer to other pr ,such as https://github.com/cloudwego/hertz/pull/169

Reply:
@li-jin-gou  Thanks, i have changed the title